DPPP/test/unit/tPSet.cc

```diff
@@ -277,58 +257,44 @@ namespace DP3 {
         BOOST_CHECK_EQUAL (pset.itsTimes[7], 24*3600+1);
       }
       {
-        ParameterSet parset;
+        DP3::ParameterSet parset;
         parset.add ("timeslot", "[2..4, 10]");
-        PreFlagger::PSet pset (in, parset, "");
+        PreFlagger::PSet pset (in.get(), parset, "");
         pset.updateInfo (in->getInfo());
         BOOST_CHECK_EQUAL (pset.itsTimeSlot.size(), size_t{4});
         BOOST_CHECK_EQUAL (pset.itsTimeSlot[0], 2u);
         BOOST_CHECK_EQUAL (pset.itsTimeSlot[1], 3u);
         BOOST_CHECK_EQUAL (pset.itsTimeSlot[2], 4u);
         BOOST_CHECK_EQUAL (pset.itsTimeSlot[3], 10u);
       }
-      // Some erronous ones.
-      bool err = false;
-      try {
-        ParameterSet parset;
+      {
+        DP3::ParameterSet parset;
         parset.add ("reltime", "[12:00:00]");
```

Comment:
I think all of these blocks should be separate tests, especially the 3 cases of the exception cases since they are exactly the same except for one input